xmodule/video_block/transcripts_utils.py

```diff
@@ -1072,9 +1077,71 @@ def get_transcript_from_learning_core(video_block, language, output_format, tran
     Returns:
         tuple containing content, filename, mimetype
     """
-    # TODO: Update to use Learning Core data models once static assets support
-    # has been added.
-    raise NotFoundError("No transcript - transcripts not supported yet by learning core components.")
+    usage_key = video_block.scope_ids.usage_id
```

Comment:
```suggestion
    usage_key = video_block.usage_key
```
[nit] you can use the shorthand now if you'd like

Reply:
Oh right, I forgot this was added to the API. Thank you.